downstream/modules/platform/con_storage-options-for-operator-installation-on-ocp.adoc

```diff
@@ -0,0 +1,14 @@
+[id="con_storage-options-for-operator-installation-on-ocp_{context}"]
+= Storage options for {OperatorPlatform} installation on {OCP}
+
+If you are using file-based storage and your installation scenario includes {HubName}, ensure that you change the `ReadWriteOnce` default storage option for {OperatorPlatform} to `ReadWriteMany`.
```

Comment:
```suggestion
If your installation includes {HubName}, you must provision a persistent volume claim with `ReadWriteOnce` access.
```

Reply:
something along those lines imho

Reply:
Actually, the goal with this is to provision with `ReadWriteMany` access mode. However, I do like the concise way of stating this, which is similar to my original statement. I updated the statement because of a peer review comment that I received previously that it might be good to mention the default option so that the customer can distinguish the two, I guess. I'll return to the original statement that keeps things simplified.